packages/odyssey-react-mui/src/theme/spacing.ts

```diff
@@ -11,17 +11,22 @@
  */
 
 import type { ThemeOptions } from "@mui/material";
+import { TokensOverride } from ".";
 import * as Tokens from "@okta/odyssey-design-tokens";
 
-export const spacing: ThemeOptions["spacing"] = [
-  0,
-  Tokens.SpaceScale0,
-  Tokens.SpaceScale1,
-  Tokens.SpaceScale2,
-  Tokens.SpaceScale3,
-  Tokens.SpaceScale4,
-  Tokens.SpaceScale5,
-  Tokens.SpaceScale6,
-  Tokens.SpaceScale7,
-  Tokens.SpaceScale8,
-];
+export const spacing = (
+  odysseyTokens: TokensOverride
+): ThemeOptions["spacing"] => {
```

Comment:
These aren't `odysseyTokens`, they're `tokensOverride` right?

Reply:
This is actually the entire odysseyTokens as we are now passing the merged tokens+overrides to the theme - https://github.com/okta/odyssey/pull/1797/files#diff-74f2cc0542c96af5eba6c163e87b573c8675e2d399008ff23825e2dd5682fc57R30